packages/e2e/scripts/local.runner.js

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+const Nightwatch = require('nightwatch');
+const browserstack = require('browserstack-local');
+let bs_local;
+
+try {
+
+  process.mainModule.filename = "./node_modules/nightwatch/bin/nightwatch"
+  // Code to start browserstack local before start of test
+  console.log("Connecting local");
+  Nightwatch.bs_local = bs_local = new browserstack.Local();
+  bs_local.start({'key': process.env.BROWSERSTACK_ACCESS_KEY }, (error) => {
```

Comment:
What if you don't have this variable in your env? Or is it guaranteed?

Reply:
i will update the docs